.gitignore

```diff
@@ -92,6 +92,9 @@ coverage-html/
 TAGS
 tags
 cscope.*
+GPATH
+GRTAGS
+GTAGS
```

Comment:
This should go into a separate commit, it has nothing to do with the change below.